src/index.js

```diff
@@ -10,17 +10,37 @@ const token = require('./helpers/jwt');
 
 const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 const JWT_SECRET = process.env.JWT_SECRET;
+const WEBHOOK_SECRET = process.env.WEBHOOK_SECRET;
+const DEBUG = process.env.DEBUG;
 
 const returnErrorResponse = message => Promise.resolve(http.response(message, http.STATUS_CODE.ERROR));
 
 async function lambda(event) {
+  if (DEBUG) console.log(event);
```

Comment:
Relying on a logging library might be beneficial if this grows, and even `console.log` can be relied on to define which level of logging we want. Nothing important at this stage.